sopt/src/test/scala/dagr/sopt/cmdline/ClpReflectiveBuilderTest.scala

```diff
@@ -112,4 +115,28 @@ class ClpReflectiveBuilderTest extends UnitSpec {
       an[IllegalStateException] should be thrownBy _.toCommandLineString
     }
   }
+
+  it should "print v if the argument type was an Option and the value was Some(v)" in {
+    val t = new ClpReflectiveBuilder(classOf[DefaultWithSomeOption])
+    t.argumentLookup.view.foreach {
+      _.toCommandLineString shouldBe "--w 4"
+    }
+  }
+
+  it should "print the special empty or none token for a None value or empty collections" in {
```

Comment:
This doesn't sit right with me.  I would think that for `None` and empty collections we should just omit those arguments.  I could see how maybe we should print the `:none:` if the value is None/empty but the default was non-empty though.